### ISSUE TITLE
Malformed http headers

### ISSUE THREAD

**Issue by amasad** [dddomodossola/remi#235]
Was running remi behind a proxy and saw that it was failing to parse the headers. Curling the webserver locally indeed shows malformed header with repeated stuff:

```
HTTP/1.0 200 OK
Server: BaseHTTP/0.6 Python/3.7.0
Date: Fri, 31 Aug 2018 22:54:48 GMT
Set-Cookie: remi_session=0
HTTP/1.0 200 OK
Server: BaseHTTP/0.6 Python/3.7.0
Date: Fri, 31 Aug 2018 22:54:48 GMT
Content-type: text/html

```

Notice the repetition of the first 3 lines.

**Comment by amasad** [comment]
The problem is `send_response` is called twice:

https://github.com/dddomodossola/remi/blob/2576f8737c7c0dfaa44eddd7019dab9cf38b5346/remi/server.py#L341

and 

https://github.com/dddomodossola/remi/blob/2576f8737c7c0dfaa44eddd7019dab9cf38b5346/remi/server.py#L836

**Comment by amasad** [comment]
Thank you for your work. We're looking to possibly adding this to repl.it because users have been asking for gui and given we run in the browser this seems like a good fit. Look forward to the fix.

**Comment by dddomodossola** [comment]
@amasad I made a fix, removing the duplicate http send_response request. Please let me know if this solves the "malformed http header" problem.
It would be cool to have remi on repl.it, different people asked me about the possibility to use remi on web devel solutions like heroku on so on. Let me know it this will be done, I will inform my contacts. ;-)

**Comment by amasad** [comment]
awesome! can we get a release so I can test it?

**Comment by amasad** [comment]
I got something working but ran into #237. Here is a hosted remi app (without the https): http://remi-repl-demo--amasad.repl.co/

Source: https://repl.it/@amasad/remi-repl-demo

**Comment by dddomodossola** [comment]
I will make a release soon, thank you for sharing the hosted app.

**Comment by dddomodossola** [comment]
@amasad  Released version 1.2.2
